src/transformers/models/xlm_roberta/tokenization_xlm_roberta.py

```diff
@@ -252,7 +270,7 @@ def get_vocab(self):
         return vocab
 
     def _tokenize(self, text):
-        return self.sp_model.EncodeAsPieces(text)
+        return self.sp_model.encode(text, out_type=str)
```

Comment:
Is this change necessary, and does it change anything of the current behavior?

Reply:
Yes - see my initial posting here:

> In the `_tokenize(self, text)` function there was a call to `self.sp_model.EncodeAsPieces(text)`. This call does ignore the parameters for subword regularization. That is why it had to be replaced by a call to `self.sp_model.encode(text, out_type=str)`.

Without this change the parameters set in the constructor (`enable_sampling`, `nbest_size` and `alpha`) would be ignored and no augmentation would be done.

Reply:
IMO this is resolved.